docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -0,0 +1,182 @@
+NAME                READY   STATUS            PROFILES      VERSION   AGE
+openshift-gateway   5/5     ComponentsReady   ["default"]   2.4.0     107s
+```
+
+OSSM will create two new Istio deployments in the `openshift-ingress` namespace. Ensure the pods for the deployments are running. These will need to become ready for the SMCP to also go ready:
+```shell
+$ oc get deployment -n openshift-ingress
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+istio-ingressgateway       1/1     1            1           42s
+istiod-openshift-gateway   1/1     1            1           55s
+router-default             2/2     2            2           6h4m
+```
+
+Gateway API and OSSM have been installed successfully and are ready to use. We are now ready to create Gateways and HTTPRoutes.
+
+## Gateway API Example
+
+Let’s now use Gateway API to create a Gateway and Route. In this example, we will convert a simple HTTP OpenShift Route into roughly equivalent Gateway API objects (Gateway, HTTPRoute). This is an example of OpenShift Route that we will convert:
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  name: http
+  namespace: demo-app
+spec:
+  host: demo.gwapi.${DOMAIN}
+  to:
+    kind: Service
+    name: foo-app
+  port:
+    targetPort: 8080
+```
+
+First, create the Gateway with a Listener (`spec.Listeners` instance) that will listen for requests on a URL under the cluster domain. The Gateway must be created in the `openshift-ingress` namespace and specify in the `spec.gatewayClassName` the GatewayClass we created earlier.
+```yaml
+$ DOMAIN=$(oc get ingresses.config/cluster -o jsonpath={.spec.domain})
+$ oc apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: demo-gateway
+  namespace: openshift-ingress
+spec:
+  gatewayClassName: openshift-default
+  listeners:
+  - name: demo
+    hostname: "*.gwapi.${DOMAIN}"
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: All
+EOF
+```
+
+By default, Istio will [automatically provision](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment) a gateway deployment and service with the same name upon creation of this Gateway object:
+```shell
+$ oc get deployment -n openshift-ingress demo-gateway
+NAME           READY   UP-TO-DATE   AVAILABLE   AGE
+demo-gateway   1/1     1            1           25s
+$ oc get service -n openshift-ingress demo-gateway
+NAME           TYPE           CLUSTER-IP       EXTERNAL-IP                          PORT(S)                        AGE
+demo-gateway   LoadBalancer   172.30.175.176   domain.us-east-1.elb.amazonaws.com   15021:30608/TCP,80:31833/TCP   47s
+```
+
+When the Gateway from the previous step is created, the Ingress Operator will automatically create a DNSRecord with the hostname from the listener:
+```shell
+$ oc get dnsrecord -n openshift-ingress
+NAME                         AGE
+gateway-7b49f58c6-wildcard   6m11s
+```
```

Comment:
Careful about confusion between the CR and the actual DNS record.  It seems worthwhile to show the expected status for the CR:
````suggestion
When the Gateway from the previous step is created, the Ingress Operator will automatically create a DNSRecord CR with the hostname from the listener:
```console
$ oc get dnsrecord -n openshift-ingress
NAME                         AGE
gateway-7b49f58c6-wildcard   6m11s
```
The Ingress Operator will then create a DNS record using the cloud-specific API (for example, Route 53 on AWS) and update the status on the DNSRecord CR:
```console
$ oc get dnsrecord/gateway-7b49f58c6-wildcard -n openshift-ingress -o yaml
kind: DNSRecord
metadata:
  name: gateway-7b49f58c6-wildcard
  namespace: openshift-ingress
  [...]
spec:
  dnsName: '*.gwapi.${DOMAIN}.'
  targets:
  - domain.us-east-1.elb.amazonaws.com
  [...]
status:
  [...]
  zones:
  - conditions:
    - message: The DNS provider succeeded in ensuring the record
      reason: ProviderSuccess
      status: "True"
      type: Published
    dnsZone:
      tags:
        [...]
  - conditions:
    - message: The DNS provider succeeded in ensuring the record
      reason: ProviderSuccess
      status: "True"
      type: Published
    dnsZone:
      id: [...]
```
````